_includes/head.html

```diff
@@ -11,18 +13,15 @@
   <link href="http://co-up.de/atom.xml" rel="alternate" title="Blog Feed" type="application/atom+xml" />
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <meta property="og:title" content="co.up needs your help!">
+  <meta property="og:title" content="{{ page.title }}">
   <meta property="og:type" content="website">
-  <meta property="og:site_name" content="co.up Community Event Space">
-  <meta property="og:description" content="The rent is rising and without your help we'll be forced to close the space. That means no more free event space for community members or organizations. We need your support—help #saveCoUp!">
-  <meta property="og:image" content="https://co-up.de/images/save-coup-og.png">
-  <meta property="og:url" content="https://co-up.de/support-us/">
+  <meta property="og:site_name" content="{{ site.name }}">
+  <meta property="og:description" content="{% if page.description | size > 1%}{{ page.description }}{% elsif post_excerpt | size > 1 %}{{post_excerpt}}{% else %}{{site.description}}{%endif%}">
```

Comment:
where is it truncated to 50 words?

Reply:
see any blog post

Reply:
in the readme it sounds like this would be done automaticaly somehow ("By default the first 50 words will be used as a preview in social media")?

Reply:
Yes it would be done automatically

Reply:
I wrote the script, see further up where I set post_excerpt